### ISSUE TITLE
 GRANT OPTION as GRANT should not be described as a regular priv.

### ISSUE THREAD

**Issue by seiya-annie** [pingcap/docs#4791]
## Error Report

**This repository is ONLY used to solve issues related to DOCS.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. What is the URL/path of the document related to this issue?

https://docs.pingcap.com/zh/tidb/dev/privilege-management

2. How would you like to improve it?

delete grand in the table

![企业微信截图_e7ea7242-875c-443f-9661-39bec203c1ee](https://user-images.githubusercontent.com/53471087/106870333-f228e680-670b-11eb-9048-14c1d5a729e7.png)

**Comment by ti-srebot** [comment]
This issue contains Chinese, please use English.

**Comment by TomShawn** [comment]
@seiya-annie Do you mean to delete the `Grant` row?

**Comment by morgo** [comment]
The column which says `Grant` should read `Grant Option`. But usually the syntax for `GRANT` option is specified as `WITH GRANT OPTION` at the end of a `GRANT` statement, so it's better to just delete the row.

These are also out of date:
![image](https://user-images.githubusercontent.com/57982/106926466-5cbb3180-66ce-11eb-8005-56b85c473c28.png)


File was added in https://github.com/pingcap/tidb/pull/19553  and I have manually verified Shutdown, Process, Usage are also implemented correctly. Execute does not apply to TiDB. The only one missing is the index privilege, but it is not so useful. I recommend deleting this section.
